scripts/test/console_output_test.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python
+
+import os
+import sys
+import pexpect
+
+def main():
+	if len(sys.argv) == 1:
+		print ("Usage: ./scripts/test/console_output_test.py <console_path>")
+		sys.exit(1)
+
+	if not os.path.isdir("resources/test_data/tbl"):
+		print ("Cannot find resources/test_data/tbl. Are you running the test suite from the main folder of the Hyrise repository?")
+		sys.exit(1)
+
+	console_path = sys.argv[1]	
+	console = pexpect.spawn(console_path + "/hyriseConsole", timeout=15, dimensions=(1920,1080))
+
+	# Test print command
+	console.sendline("print test")
+	console.expect("Exception thrown while loading table:")
+	
+	# Test load command
+	console.sendline("load resources/test_data/tbl/10_ints.tbl test")
+	console.expect('Loading .*tbl/10_ints.tbl into table "test"')
+	console.expect('Encoding "test" using Unencoded')
```

Comment:
assume the next line that the console would write would be "Error applying encoding"  - or something to that extend. Would the test script still succeed, or would it fail?

Reply:
No, there is no method, which throws an exception, when a pattern is matched (something like not_expect()). If the encoding fails, a different message instead of ''Encoding "test" using Unencoded' has to be returned.